sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/planner/BeamRuleSets.java

```diff
@@ -20,25 +20,7 @@
 import java.util.List;
 import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.extensions.sql.impl.rel.BeamRelNode;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamAggregateProjectMergeRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamAggregationRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamBasicAggregationRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamCalcRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamCoGBKJoinRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamEnumerableConverterRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamIOPushDownRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamIntersectRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamJoinAssociateRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamJoinPushThroughJoinRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamMinusRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamSideInputJoinRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamSideInputLookupJoinRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamSortRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamTableFunctionScanRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamUncollectRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamUnionRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamUnnestRule;
-import org.apache.beam.sdk.extensions.sql.impl.rule.BeamValuesRule;
+import org.apache.beam.sdk.extensions.sql.impl.rule.*;
```

Comment:
This might be some default packing behavior of my Intellij idea. I didn't realize this happened; will unpack the import statement in later commits.

Reply:
Yes. Intellij has a setting to control when using `*`: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports/25916603